src/pages/proposal.tsx

```diff
@@ -123,6 +124,17 @@ function formatDescription(description: string) {
   return position > 0 ? value.slice(0, position).trim() : value
 }
 
+function getProposalView(proposal: ProposalAttributes | null) {
+  switch (proposal?.type) {
```

Comment:
Maybe return null if proposal is null?

Reply:
In that case, I think a "null" text would be displayed